### PR TITLE
Score-Tracker

### PR DESCRIPTION
**Pull request by Grace-Aka**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+const prompt = require('prompt-sync')();
+
+let input = prompt ('Please input a score between 0 and 100: ');
+
+function checkInput(){
+    if (input >= 0 && input <= 30 || input >= 31 && input <= 60 || input >= 61 && input <= 100  && input !='') {
```
Comment (Phildeoner) [inline script.js]: At this point you can just say 
if (input >= 0 &&  input <= 100 && input != '' )

File: script.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    } else if ( input >=61 && input <=100) {
+        console.log("Wow you did great!");
+        return input;
+    } else {
+        console.log('Please input a vaild score')
+    }
+    performOperation();
+}
+
+
```
Comment (Phildeoner) [inline script.js]: remove this space

File: script.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+checkInput();
+
+
+function performOperation(){
+    
+    if (input >=0 && input <=30) {
+        console.log("Try Harder your score is low!");
+        return input;
+
```
Comment (Phildeoner) [inline script.js]: Remove this space

File: script.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+       performOperation();
+    } else {
+        console.log('Please insert a valid score!');
+    }
+
+    resetCheckInput();
+  }
+
+checkInput();
+
```
Comment (Phildeoner) [inline script.js:14]: Remove this space

File: script.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+function performOperation(){
+    
+    if (input >=0 && input <=30) {
+        console.log("Try Harder your score is low!");
+        return input;
```
Comment (Phildeoner) [inline script.js]: Check your return I dont think it should be input but rather you can set what your result (what you are console logging) as a variable and return it, but that return is not necessary, check same for all other returns

**Review by Phildeoner — CHANGES_REQUESTED** [review]
Hi @Grace-Aka 👋,
You did a great job👍, consider going through the highlighted suggestions below, what do you think?
Happy coding😊

File: script.js (hunk 1 above)
Comment (Grace-Aka) [reply script.js]: Done.. thanks

File: script.js (hunk 5 above)
Comment (Grace-Aka) [reply script.js]: Done

File: script.js (hunk 2 above)
Comment (Grace-Aka) [reply script.js]: Done

File: script.js (hunk 3 above)
Comment (Grace-Aka) [reply script.js]: Done

**Pull request merged**